import-scripts/updated_filename_for_datatype_test.py

```diff
@@ -0,0 +1,534 @@
+meta_resource_definition.txt
+meta_resource_study.txt
+meta_resource_sample.txt
+meta_phosphoprotein_quantification.txt
+meta_mirna.txt
+meta_mirna_zscores.txt
+meta_mrna_affymetrix_microarray.txt
+meta_mrna_affymetrix_microarray_zscores_ref_diploid_samples.txt
+meta_mrna_affymetrix_microarray_zscores_ref_all_samples.txt
+meta_mrna_agilent_microarray.txt
+meta_mrna_agilent_microarray_zscores_ref_diploid_samples.txt
+meta_mrna_agilent_microarray_zscores_ref_all_samples.txt
+meta_mrna_seq_rpkm.txt
+meta_mrna_seq_rpkm_zscores_ref_diploid_samples.txt
+meta_mrna_seq_rpkm_zscores_ref_all_samples.txt
+meta_mrna_seq_v2_rsem.txt
+meta_mrna_seq_v2_rsem_zscores_ref_diploid_samples.txt
+meta_mrna_seq_v2_rsem_zscores_ref_all_samples.txt
+meta_mrna_seq_v2_rsem_normal_samples.txt
+meta_mrna_seq_v2_rsem_normal_samples_zscores_ref_normal_samples.txt
+meta_mrna_seq_v2_rsem_zscores_ref_normal_samples.txt
+meta_mrna_seq_cpm.txt
+meta_mrna_seq_cpm_zscores_ref_diploid_samples.txt
+meta_mrna_seq_cpm_zscores_ref_all_samples.txt
+meta_mrna_seq_tpm.txt
+meta_mrna_seq_tpm_zscores_ref_diploid_samples.txt
+meta_mrna_seq_tpm_zscores_ref_all_samples.txt
+meta_mrna_seq_fpkm_capture.txt
+meta_mrna_seq_fpkm_capture_zscores_ref_diploid_samples.txt
+meta_mrna_seq_fpkm_capture_zscores_ref_all_samples.txt
+meta_mrna_seq_fpkm_polya.txt
+meta_mrna_seq_fpkm_polya_zscores_ref_diploid_samples.txt
+meta_mrna_seq_fpkm_polya_zscores_ref_all_samples.txt
+meta_mrna_mirna_merged_zscores.txt
+meta_mrna_affymetrix_microarray_merged.txt
+meta_mrna_affymetrix_microarray_merged_zscores_ref_diploid_samples.txt
+meta_mrna_outliers.txt
+meta_rppa_zscores.txt
+meta_protein_quantification_zscores.txt
+meta_cna_foundation.txt
+meta_cna_rae.txt
+meta_cna_consensus.txt
+meta_cna.txt
+meta_linear_cna.txt
+meta_log2_cna.txt
+meta_armlevel_cna.txt
+meta_drug_treatment_auc.txt
+meta_drug_treatment_ic50.txt
+meta_drug_treatment_zscore.txt
+meta_mutations_uniprot_canonical_transcripts.txt
+meta_mutations_uniprot_canonical_transcripts.txt
+meta_mutations_uniprot_canonical_transcripts.txt
+meta_gene_matrix.txt
+meta_gene_panel_matrix.txt
+"""
+
+def filename_maps():
+    datatype_lines = datatype_column_content().splitlines()
+    staging_file_lines = staging_filename_column_content().splitlines()
+    meta_file_lines = meta_filename_column_content().splitlines()
+    if len(datatype_lines) != len(staging_file_lines):
+        print("Error : datatype_column_content linecount (%d) does not match staging_filename_column_content linecount (%d)" % (len(datatype_lines), len(staging_file_lines)))
+        sys.exit(1);
+    if len(datatype_lines) != len(meta_file_lines):
+        print("Error : datatype_column_content linecount (%d) does not match meta_filename_column_content linecount (%d)" % (len(datatype_lines), len(meta_file_lines)))
+        sys.exit(1);
+    filename2datatype = {}
+    datatype_seen = set([])
+    filename_is_first_listed = set([])
+    for index in range(1,len(datatype_lines)):
```

Comment:
if we're looping through items in the list:
`for x in list:`

Reply:
looping on index number removed